core/src/style/polygonStyle.cpp

```diff
@@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-VboMesh* PolygonStyle::newMesh() const {
-    return new Mesh(m_vertexLayout, m_drawMode);
-}
+namespace {
```

Comment:
One consequence of using anonymous namespaces for these structs is that I can't seem to differentiate them in a profiling context, which makes it hard to evaluate their individual performance.